### PR TITLE
Restructured BetaFlight controller into BetaAviary and CTBRControl. P…

### PR DESCRIPTION
**Pull request by JacopoPan**
…rovided beta2 example script.

known issue: slight variations exist between flight results of beta.py and beta2.py

### CODE REVIEW

**Review by JacopoPan — COMMENTED** [review]
LGTM, I will be adding UDP port changes and automation scripts to support multiple SITL in the upcoming commits

**Comment by JacopoPan** [comment]
TODOs (CC @spencerteetaert)
- modify `BetaAviary` to support multiple SITL betaflights on different UDP ports
- automate the clone/edit/compile/config/launch of multiple SITL beta flights

File: README.md
[the remark this answers is not in the document]
Comment (spencerteetaert) [reply README.md:1]: Instructions clearer in beta.py. Approved, maybe remove the duplicate instructions and reference one from another

**Review by spencerteetaert — APPROVED** [review]

**Comment by JacopoPan** [comment]
@spencerteetaert 

multi BF example now working fine, only missing adding README/docstrings instructions on how to use `assets/clone_bfs.sh`

**Pull request merged**